src/stan/math/prim/scal/prob/neg_binomial_2_log_rng.hpp

```diff
@@ -31,20 +31,45 @@ namespace stan {
                            RNG& rng) {
       using boost::variate_generator;
       using boost::random::negative_binomial_distribution;
+      using boost::random::poisson_distribution;
+      using boost::gamma_distribution;
 
       static const char* function("stan::math::neg_binomial_2_log_rng");
+      static const double POISSON_MAX_RATE = std::pow(2.0, 30);
 
       using stan::math::check_finite;
```

Comment:
we can remove all these `using stan::math::check_*` statements: this is all in the same namespace now.